src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -313,6 +318,10 @@ class FileField extends React.Component {
                 setTimeout(() => {
                   focusElement(`[name="get_password_${index}"]`);
                 }, 100);
+              } else if (hasErrors && enableShortWorkflow) {
```

Comment:
The retry button only exist when the `enableShortWorkflow` flag is on.

Reply:
Why are we wrapping the focus call in a timeout?

Reply:
I needed the focus to happen after the button is rendered. Was going to use a hook at first, but saw that the focus for the password input was wrapped in a timeout, and figured it must be for a good reason, so I followed that pattern.